src/components/Game.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import Board from './Board'
+
+//component
+const Game: React.FC = () => {
```

Comment:
```suggestion
const Game = () => {
```

let TypeScript implicitly determine the return types for React components